### ISSUE TITLE
map name filter in the server browser

### ISSUE THREAD

**Issue by ETLegacyTeam** [etlegacy/etlegacy#314]
Author: TunPal (TunPal)
Date: 2013-06-10
Redmine Issue: 314, https://dev.etlegacy.com/issues/314

---

add mapname filter (with partial name match) in the server browser

---

Related commit(s):
ac9586ae701699f59625510d50027ba660ce564c - ui: simple fix, refs 314
 93a28eb1376a7945dc20039149bcc58de593b3fe - ui: simple fix, refs 314
 0956c04d5ccf3ffc797a930a00145111bede6e69 - ui: legacy mod added to mod filter refs 314
 49511007f874285363ab44f23074175c81319039 - ui: reset filters button added fixes 338 & filter..

**Comment by ETLegacyTeam** [comment]
From: kkthx (kkthx)
Date: 2013-09-01T18:27:54Z

---

Feature applied in commit:5eaf4ae4a251bbef3a88a097bb01cd08bbb77d91

**Comment by ETLegacyTeam** [comment]
From: @IR4T4 (IR4T4)
Date: 2013-09-01T21:23:58Z

---

There is an issue with the line 185 (playonline.menu):
EDITFIELDSIMPLE( 8+.5*((FILTERS_WIDTH)-6)+16+14, 140, .5*((FILTERS_WIDTH)-6)-48, 10, "", .2, 8, "ui_browserMapFilter", 25, 25, "Includes or excludes servers with desired map name, from the server list" )

See console
```
ERROR: ui/playonline.menu, line 185: unknown menu item keyword *
ERROR: ui/playonline.menu, line 185: unknown menu item keyword (
ERROR: ui/playonline.menu, line 185: unknown menu item keyword (
ERROR: ui/playonline.menu, line 185: unknown menu item keyword .75
ERROR: ui/playonline.menu, line 185: unknown menu item keyword *
ERROR: ui/playonline.menu, line 185: unknown menu item keyword (
ERROR: ui/playonline.menu, line 185: unknown menu item keyword 608
ERROR: ui/playonline.menu, line 185: unknown menu item keyword -
ERROR: ui/playonline.menu, line 185: unknown menu item keyword 18
ERROR: ui/playonline.menu, line 185: unknown menu item keyword )
ERROR: ui/playonline.menu, line 185: unknown menu item keyword )
ERROR: ui/playonline.menu, line 185: unknown menu item keyword -
ERROR: ui/playonline.menu, line 185: unknown menu item keyword 6
ERROR: ui/playonline.menu, line 185: unknown menu item keyword )
ERROR: ui/playonline.menu, line 185: unknown menu item keyword -
ERROR: ui/playonline.menu, line 185: unknown menu item keyword 48

```

**Comment by ETLegacyTeam** [comment]
From: kkthx (kkthx)
Date: 2013-09-01T23:02:45Z

---

I think everything seems to work fine - my console does not print any errors

See https://github.com/etlegacy/etlegacy/blob/master/etmain/ui/playonline.menu#L185

**Comment by ETLegacyTeam** [comment]
From: @jackeri (Jacker)
Date: 2013-09-02T16:55:10Z

---

One thing is that, if you enable the map filter and afterwards edit the text line (or write it after the enable action) it wont update the list, you need to re-enable the filter if you modify the text.

**Comment by ETLegacyTeam** [comment]
From: @jackeri (Jacker)
Date: 2013-09-03T14:18:04Z

---

This feature now works properly.
Fixed on: commit:edcd192b47ef6bc7e03db0906bd2a944a811763a
